app/logic/Mail/EWS/EWSAccount.ts

```diff
@@ -360,7 +360,7 @@ export class EWSAccount extends MailAccount {
         m$ConnectionTimeout: 29, // minutes
       },
     };
-    this.callStream(streamRequest, async message => {
+    await this.callStream(streamRequest, async message => {
```

Comment:
This was deliberately not an `await`, because it takes 30 minutes to complete.

Reply:
Ah, thanks for the note. We'll change that to `this.callStream(...).catch(this.errorCallback);`

@NeilRashbrook  Having neither of those is *always* a bug. It leads to uncaught exceptions, and even to crashes. So, every `async` function call (and every promise run) *must* be either with `await` or with `.catch()`.

Reply:
`callStream` already has its own `try/catch`.

Reply:
Fixed in git commit 9cd12445

Reply:
@NeilRashbrook Thanks for alerting about this. Indeed, `await` for 29 minutes would have been bad here.